alpenhorn/common/config.py

```diff
@@ -194,21 +209,15 @@ def load_config():
             config = merge_dict_tree(config, conf)
 
     if not any_exist:
-        raise RuntimeError("No configuration files available.")
-
-
-class ConfigClass(object):
```

Comment:
I've deleted this vestigial thing that seemingly was never properly implemented.  (The `config` module was the one module that escaped the Great Refactoring or this probably would have been deleted then.)